src/pages/observationPages/ObservationFormPageDefault.js

```diff
@@ -0,0 +1,186 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import { v4 as uuidv4 } from 'uuid'
+import Button from '../../components/buttons/Button'
+import useFormValidation from '../../hooks/useFormValidation'
+import getCurrentTime from '../../utils/getCurrentTime'
+
+ObservationFormPageDefault.propTypes = {
+  observationTypes: PropTypes.arrayOf(
+    PropTypes.shape({
+      name: PropTypes.string,
+      type: PropTypes.string,
+      unit: PropTypes.string,
+    }).isRequired
+  ),
+  observationType: PropTypes.string,
+  onSubmit: PropTypes.func.isRequired,
+}
+
+export default function ObservationFormPageDefault({
+  observationTypes,
+  observationType,
+  onSubmit,
+}) {
+  let history = useHistory()
+  const { name, type, unit, format } = observationTypes.find(
+    element => element.type === observationType
+  )
+
+  const [inputs, setInputs] = useState({
+    time: getCurrentTime(),
+    inputValue: '',
+  })
+
+  const {
+    isTimeValid,
+    isDisabled,
+    setIsTimeValid,
+    validateTime,
+    validateTypeInput,
+    isObservationInputValid,
+    setIsObservationInputValid,
+  } = useFormValidation(inputs)
+
+  return (
+    <FormWrapper
+      onSubmit={handleSubmit}
+      aria-label="dynamisch erstellen"
+      role="form"
+    >
+      <Label>
+        {name}:
+        <InputWrapper>
+          <Input
+            name="inputValue"
+            onChange={handleChange}
+            isInputValid={isObservationInputValid}
+            value={inputs.inputValue}
+            autoComplete="off"
+          />
+          <Unit>{unit}</Unit>
+        </InputWrapper>
+        <Warning isInputValid={isObservationInputValid}>
+          {name} bitte in einem Format wie "{format}" angeben!
+        </Warning>
+      </Label>
+      <Label>
+        Uhrzeit:
+        <InputWrapper>
+          <Input
+            name="time"
+            onChange={handleChange}
+            isInputValid={isTimeValid}
+            value={inputs.time}
+            autoComplete="off"
+          />
+          <Unit>Uhr</Unit>
+        </InputWrapper>
+        <Warning isInputValid={isTimeValid}>
+          Bitte gib eine Uhrzeit im Format h:mm oder hh:mm an!
+        </Warning>
+      </Label>
+      <Flexbox>
+        <Button onClick={handleBackClick} type="button">
+          zurück
+        </Button>
+        <Button disabled={isDisabled}>speichern</Button>
+      </Flexbox>
+    </FormWrapper>
+  )
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    if (isDisabled) return
+    const form = event.target
```

Comment:
Just a small suggestion but I think it would be nicer to read if there is one line between these 🙂 
```suggestion
    if (isDisabled) return
    
    const form = event.target
```